WikiWatershed/gwlf-e
# Get correct model output for hydrology

The hydrology output from this port of GWLF-E was incorrect.  This PR gets the values to align, with some minor rounding differences, to the original vb6 implementation.   Fixes include:

- Fix indentation for monthly loop that was included in daily loop
- Don't use `-=` syntax, which has subtly different order of operations than the straight vb syntax
- Include missing conditional and assignment

These changes align all output fields (except AvPrecip which was already correct) but may not guarantee that the sedimentation values yet to be ported are correct.

Connects https://github.com/WikiWatershed/gwlf-e/issues/38

#### To test:
Ideally one would check my changes against the VB code, but it is sufficient to run the model on `/testinput_4.gms` and then use the `compare_output` script, as referenced in `README` to check against the known values from the compiled VB model.
